salt/cloud/clouds/digital_ocean.py

```diff
@@ -845,7 +845,12 @@ def destroy_dns_records(fqdn):
     '''
     domain = '.'.join(fqdn.split('.')[-2:])
     hostname = '.'.join(fqdn.split('.')[:-2])
-    response = query(method='domains', droplet_id=domain, command='records')
+    # TODO: remove this when the todo on 754 is available
+    try:
+        response = query(method='domains', droplet_id=domain, command='records')
+    except SaltCloudSystemExit
+        log.debug('Failed to find domains.')
```

Comment:
Do we really only want to log this at a debug level and then return? That might make debugging hard when the response fails legitimately.

This also brings up another point - do we really want to be calling the `destroy_dns_records